libgit/init.go

```diff
@@ -37,29 +43,44 @@ func Params(kbCtx libkbfs.Context, storageRoot string) (
 		return libkbfs.InitParams{}, "", err
 	}
 
-	params = libkbfs.DefaultInitParams(kbCtx)
+	if paramsBase != nil {
+		params = *paramsBase
+	} else {
+		params = libkbfs.DefaultInitParams(kbCtx)
+	}
 	params.LogToFile = true
 	// Set the debug default to true only if the env variable isn't
 	// explicitly set to a false option.
 	envDebug := os.Getenv("KBFSGIT_DEBUG")
 	if envDebug != "0" && envDebug != "false" && envDebug != "no" {
 		params.Debug = true
 	}
+	// This is set to false in docker tests for now, but we need it. So
+	// override it to true here.
+	params.EnableJournal = true
 	params.EnableDiskCache = false
 	params.StorageRoot = tempDir
 	params.Mode = libkbfs.InitSingleOpString
 	params.TLFJournalBackgroundWorkStatus =
 		libkbfs.TLFJournalSingleOpBackgroundWorkEnabled
+
+	if baddr := os.Getenv(paramKeybaseGitBServerAddr); len(baddr) > 0 {
```

Comment:
Why do we need this if we're passing in the params explicitly?

Reply:
This is for the `git-remote-keybase` binary. The explicitly passed-in params is for the RPC handler that lives inside the `kbfsfuse` binary, which serves git related RPCs with a separate config (`gitConfig`).